Build/Projects/FrameworkReferences.Net.definition

```diff
@@ -7,12 +7,14 @@
     <Reference Include="System.Core" />
     <Reference Include="Xamarin.iOS" />
     <Reference Include="Lidgren.Network" />
+    <Reference Include="MonoGame.Framework.Core" />
```

Comment:
As @hach-que noted later @tomspilman this comment and other related one's we recanted, because Linux, Web and WP8 are excluded from this, we can't make it global.